lib/publiccloud/utils.pm

```diff
@@ -163,7 +163,7 @@ sub is_hardened() {
 
 sub is_embargo_update {
     my ($incident) = @_;
-    script_retry("curl -sf https://build.suse.de/attribs/SUSE:Maintenance:$incident -o /tmp/$incident.txt");
+    script_retry("curl -f https://build.suse.de/attribs/SUSE:Maintenance:$incident -o /tmp/$incident.txt");
```

Comment:
no please , things like this really annoys me in our logs : 
```
Alive[                                                                ]  0.0130%
Alive[                                                                ]  0.0260%
Alive[                                                                ]  0.0391%
Alive[                                                                ]  0.0521%
Alive[                                                                ]  0.0651%
Alive[                                                                ]  0.0781%
Alive[                                                                ]  0.0911%
Alive[                                                                ]  0.1042%
Alive[                                                                ]  0.1172%
Alive[                                                                ]  0.1302%
Alive[                                                                ]  0.1432%
Alive[                                                                ]  0.1562%
Alive[                                                                ]  0.1693%
Alive[                                                                ]  0.1823%
Alive[                                                                ]  0.1953%
Alive[                                                                ]  0.2083%
Alive[                                                                ]  0.2214%
Alive[                                                                ]  0.2344%
Alive[                                                                ]  0.2474%
Alive[                                                                ]  0.2604%
Alive[                                                                ]  0.2734%
Alive[                                                                ]  0.2865%
Alive[                                                                ]  0.2995%
Alive[                                                                ]  0.3125%
Alive[                                                                ]  0.3255%
Alive[                                                                ]  0.3385%
Alive[                                                                ]  0.3516%
Alive[                                                                ]  0.3646%
Alive[                                                                ]  0.3776%
Alive[                                                                ]  0.3906%
Alive[                                                                ]  0.4036%
Alive[                                                                ]  0.4167%
Alive[                                                                ]  0.4297%
Alive[                                                                ]  0.4427%
Alive[                                                                ]  0.4557%
Alive[                                                                ]  0.4687%
Alive[                                                                ]  0.4818%
Alive[                                                                ]  0.4948%
Alive[                                                                ]  0.5078%
Alive[                                                                ]  0.5208%
Alive[                                                                ]  0.5339%
Alive[                                                                ]  0.5469%
Alive[                                                                ]  0.5599%
Alive[                                                                ]  0.5729%
Alive[                                                                ]  0.5859%
Alive[                                                                ]  0.5990%
Alive[                                                                ]  0.6120%
Alive[                                                                ]  0.6250%
Alive[                                                                ]  0.6380%
Alive[                                                                ]  0.6510%
Alive[                                                                ]  0.6641%
Alive[                                                                ]  0.6771%
Alive[                                                                ]  0.6901%
Alive[                                                                ]  0.7031%
Alive[                                                                ]  0.7161%
Alive[                                                                ]  0.7292%
Alive[                                                                ]  0.7422%
Alive[                                                                ]  0.7552%
Alive[                                                                ]  0.7682%
Alive[                                                                ]  0.7812%
Alive[                                                                ]  0.7943%
Alive[                                                                ]  0.8073%
Alive[                                                                ]  0.8203%
Alive[                                                                ]  0.8333%
Alive[                                                                ]  0.8464%
Alive[                                                                ]  0.8594%
Alive[                                                                ]  0.8724%
Alive[                                                                ]  0.8854%
Alive[                                                                ]  0.8984%
Alive[                                                                ]  0.9115%
Alive[                                                                ]  0.9245%
Alive[                                                                ]  0.9375%
Alive[                                                                ]  0.9505%
Alive[                                                                ]  0.9635%
Alive[                                                                ]  0.9766%
Alive[                                                                ]  0.9896%
Alive[                                                                ]  1.0026%
Alive[                                                                ]  1.0156%
Alive[                                                                ]  1.0286%
Alive[                                                                ]  1.0417%
Alive[                                                                ]  1.0547%
Alive[                                                                ]  1.0677%
Alive[                                                                ]  1.0807%
Alive[                                                                ]  1.0937%
Alive[                                                                ]  1.1068%
Alive[                                                                ]  1.1198%
Alive[                                                                ]  1.1328%
Alive[                                                                ]  1.1458%
Alive[                                                                ]  1.1589%
Alive[                                                                ]  1.1719%
Alive[                                                                ]  1.1849%
Alive[                                                                ]  1.1979%
Alive[                                                                ]  1.2109%
Alive[                                                                ]  1.2240%
Alive[                                                                ]  1.2370%
Alive[                                                                ]  1.2500%
Alive[                                                                ]  1.2630%
Alive[                                                                ]  1.2760%
Alive[                                                                ]  1.2891%
Alive[                                                                ]  1.3021%
Alive[                                                                ]  1.3151%
Alive[                                                                ]  1.3281%
Alive[                                                                ]  1.3411%
Alive[                                                                ]  1.3542%
Alive[                                                                ]  1.3672%
Alive[                                                                ]  1.3802%
Alive[                                                                ]  1.3932%
Alive[                                                                ]  1.4062%
Alive[                                                                ]  1.4193%
Alive[                                                                ]  1.4323%
Alive[                                                                ]  1.4453%
Alive[                                                                ]  1.4583%
Alive[                                                                ]  1.4714%
Alive[                                                                ]  1.4844%
Alive[                                                                ]  1.4974%
Alive[                                                                ]  1.5104%
Alive[                                                                ]  1.5234%
Alive[                                                                ]  1.5365%
Alive[                                                                ]  1.5495%
Alive[                                                                ]  1.5625%
Alive[#                                                               ]  1.5755%
Alive[#                                                               ]  1.5885%
Alive[#                                                               ]  1.6016%
Alive[#                                                               ]  1.6146%
Alive[#                                                               ]  1.6276%
Alive[#                                                               ]  1.6406%
Alive[#                                                               ]  1.6536%
Alive[#                                                               ]  1.6667%
Alive[#                                                               ]  1.6797%
Alive[#                                                               ]  1.6927%
Alive[#                                                               ]  1.7057%
Alive[#                                                               ]  1.7187%
Alive[#                                                               ]  1.7318%
Alive[#                                                               ]  1.7448%
Alive[#                                                               ]  1.7578%
Alive[#                                                               ]  1.7708%
Alive[#                                                               ]  1.7839%
Alive[#                                                               ]  1.7969%
Alive[#                                                               ]  1.8099%
Alive[#                                                               ]  1.8229%
Alive[#                                                               ]  1.8359%
Alive[#                                                               ]  1.8490%
Alive[#                                                               ]  1.8620%
Alive[#                                                               ]  1.8750%
Alive[#                                                               ]  1.8880%
Alive[#                                                               ]  1.9010%
Alive[#                                                               ]  1.9141%
Alive[#                                                               ]  1.9271%
Alive[#                                                               ]  1.9401%
Alive[#                                                               ]  1.9531%
Alive[#                                                               ]  1.9661%
Alive[#                                                               ]  1.9792%
Alive[#                                                               ]  1.9922%
Alive[#                                                               ]  2.0052%
Alive[#                                                               ]  2.0182%
Alive[#                                                               ]  2.0312%
Alive[#                                                               ]  2.0443%
Alive[#                                                               ]  2.0573%
Alive[#                                                               ]  2.0703%
Alive[#                                                               ]  2.0833%
Alive[#                                                               ]  2.0964%
Alive[#                                                               ]  2.1094%
Alive[#                                                               ]  2.1224%
Alive[#                                                               ]  2.1354%
Alive[#                                                               ]  2.1484%
Alive[#                                                               ]  2.1615%
Alive[#                                                               ]  2.1745%
Alive[#                                                               ]  2.1875%
Alive[#                                                               ]  2.2005%
Alive[#                                                               ]  2.2135%
Alive[#                                                               ]  2.2266%
Alive[#                                                               ]  2.2396%
Alive[#                                                               ]  2.2526%
Alive[#                                                               ]  2.2656%
Alive[#                                                               ]  2.2786%
Alive[#                                                               ]  2.2917%
Alive[#                                                               ]  2.3047%
Alive[#                                                               ]  2.3177%
Alive[#                                                               ]  2.3307%
Alive[#                                                               ]  2.3437%
Alive[#                                                               ]  2.3568%
Alive[#                                                               ]  2.3698%
Alive[#                                                               ]  2.3828%
Alive[#                                                               ]  2.3958%
Alive[#                                                               ]  2.4089%
Alive[#                                                               ]  2.4219%
Alive[#                                                               ]  2.4349%
Alive[#                                                               ]  2.4479%
Alive[#                                                               ]  2.4609%
Alive[#                                                               ]  2.4740%
Alive[#                                                               ]  2.4870%
Alive[#                                                               ]  2.5000%
Alive[#                                                               ]  2.5130%
Alive[#                                                               ]  2.5260%
Alive[#                                                               ]  2.5391%
Alive[#                                                               ]  2.5521%
Alive[#                                                               ]  2.5651%
Alive[#                                                               ]  2.5781%
Alive[#                                                               ]  2.5911%
Alive[#                                                               ]  2.6042%
Alive[#                                                               ]  2.6172%
Alive[#                                                               ]  2.6302%
Alive[#                                                               ]  2.6432%
Alive[#                                                               ]  2.6562%
Alive[#                                                               ]  2.6693%
Alive[#                                                               ]  2.6823%
Alive[#                                                               ]  2.6953%
Alive[#                                                               ]  2.7083%
Alive[#                                                               ]  2.7214%
Alive[#                                                               ]  2.7344%
Alive[#                                                               ]  2.7474%
Alive[#                                                               ]  2.7604%
Alive[#                                                               ]  2.7734%
Alive[#                                                               ]  2.7865%
Alive[#                                                               ]  2.7995%
Alive[#                                                               ]  2.8125%
Alive[#                                                               ]  2.8255%
Alive[#                                                               ]  2.8385%
Alive[#                                                               ]  2.8516%
Alive[#                                                               ]  2.8646%
Alive[#                                                               ]  2.8776%
Alive[#                                                               ]  2.8906%
Alive[#                                                               ]  2.9036%
Alive[#                                                               ]  2.9167%
Alive[#                                                               ]  2.9297%
Alive[#                                                               ]  2.9427%
Alive[#                                                               ]  2.9557%
Alive[#                                                               ]  2.9687%
Alive[#                                                               ]  2.9818%
Alive[#                                                               ]  2.9948%
Alive[#                                                               ]  3.0078%
Alive[#                                                               ]  3.0208%
Alive[#                                                               ]  3.0339%
Alive[#                                                               ]  3.0469%
Alive[#                                                               ]  3.0599%
Alive[#                                                               ]  3.0729%
Alive[#                                                               ]  3.0859%
Alive[#                                                               ]  3.0990%
Alive[#                                                               ]  3.1120%
Alive[#                                                               ]  3.1250%
Alive[##                                                              ]  3.1380%
Alive[##                                                              ]  3.1510%
Alive[##                                                              ]  3.1641%
Alive[##                                                              ]  3.1771%
Alive[##                                                              ]  3.1901%
Alive[##                                                              ]  3.2031%
Alive[##                                                              ]  3.2161%
Alive[##                                                              ]  3.2292%
Alive[##                                                              ]  3.2422%
Alive[##                                                              ]  3.2552%
Alive[##                                                              ]  3.2682%
Alive[##                                                              ]  3.2812%
Alive[##                                                              ]  3.2943%
Alive[##                                                              ]  3.3073%
Alive[##                                                              ]  3.3203%
Alive[##                                                              ]  3.3333%
Alive[##                                                              ]  3.3464%
Alive[##                                                              ]  3.3594%
Alive[##                                                              ]  3.3724%
Alive[##                                                              ]  3.3854%
Alive[##                                                              ]  3.3984%
Alive[##                                                              ]  3.4115%
Alive[##                                                              ]  3.4245%
Alive[##                                                              ]  3.4375%
Alive[##                                                              ]  3.4505%
Alive[##                                                              ]  3.4635%
Alive[##                                                              ]  3.4766%
Alive[##                                                              ]  3.4896%
Alive[##                                                              ]  3.5026%
Alive[##                                                              ]  3.5156%
Alive[##                                                              ]  3.5286%
Alive[##                                                              ]  3.5417%
Alive[##                                                              ]  3.5547%
Alive[##                                                              ]  3.5677%
Alive[##                                                              ]  3.5807%
Alive[##                                                              ]  3.5937%
Alive[##                                                              ]  3.6068%
Alive[##                                                              ]  3.6198%
Alive[##                                                              ]  3.6328%
Alive[##                                                              ]  3.6458%
Alive[##                                                              ]  3.6589%
Alive[##                                                              ]  3.6719%
Alive[##                                                              ]  3.6849%
Alive[##                                                              ]  3.6979%
Alive[##                                                              ]  3.7109%
Alive[##                                                              ]  3.7240%
Alive[##                                                              ]  3.7370%
Alive[##                                                              ]  3.7500%
Alive[##                                                              ]  3.7630%
Alive[##                                                              ]  3.7760%
Alive[##                                                              ]  3.7891%
Alive[##                                                              ]  3.8021%
Alive[##                                                              ]  3.8151%
Alive[##                                                              ]  3.8281%
Alive[##                                                              ]  3.8411%
Alive[##                                                              ]  3.8542%
Alive[##                                                              ]  3.8672%
Alive[##                                                              ]  3.8802%
Alive[##                                                              ]  3.8932%
Alive[##                                                              ]  3.9062%
Alive[##                                                              ]  3.9193%
Alive[##                                                              ]  3.9323%
Alive[##                                                              ]  3.9453%
Alive[##                                                              ]  3.9583%
Alive[##                                                              ]  3.9714%
Alive[##                                                              ]  3.9844%
Alive[##                                                              ]  3.9974%
Alive[##                                                              ]  4.0104%
Alive[##                                                              ]  4.0234%
Alive[##                                                              ]  4.0365%
Alive[##                                                              ]  4.0495%
Alive[##                                                              ]  4.0625%
Alive[##                                                              ]  4.0755%
Alive[##                                                              ]  4.0885%
Alive[##                                                              ]  4.1016%
Alive[##                                                              ]  4.1146%
Alive[##                                                              ]  4.1276%
Alive[##                                                              ]  4.1406%
Alive[##                                                              ]  4.1536%
Alive[##                                                              ]  4.1667%
Alive[##                                                              ]  4.1797%
Alive[##                                                              ]  4.1927%
Alive[##                                                              ]  4.2057%
Alive[##                                                              ]  4.2187%
Alive[##                                                              ]  4.2318%
Alive[##                                                              ]  4.2448%
Alive[##                                                              ]  4.2578%
Alive[##                                                              ]  4.2708%
Alive[##                                                              ]  4.2839%
Alive[##                                                              ]  4.2969%
Alive[##                                                              ]  4.3099%
Alive[##                                                              ]  4.3229%
Alive[##                                                              ]  4.3359%
Alive[##                                                              ]  4.3490%
Alive[##                                                              ]  4.3620%
Alive[##                                                              ]  4.3750%
Alive[##                                                              ]  4.3880%
Alive[##                                                              ]  4.4010%
Alive[##                                                              ]  4.4141%
Alive[##                                                              ]  4.4271%
Alive[##                                                              ]  4.4401%
Alive[##                                                              ]  4.4531%
Alive[##                                                              ]  4.4661%
Alive[##                                                              ]  4.4792%
Alive[##                                                              ]  4.4922%
Alive[##                                                              ]  4.5052%
Alive[##                                                              ]  4.5182%
Alive[##                                                              ]  4.5312%
Alive[##                                                              ]  4.5443%
Alive[##                                                              ]  4.5573%
Alive[##                                                              ]  4.5703%
Alive[##                                                              ]  4.5833%
Alive[##                                                              ]  4.5964%
Alive[##                                                              ]  4.6094%
Alive[##                                                              ]  4.6224%
Alive[##                                                              ]  4.6354%
Alive[##                                                              ]  4.6484%
Alive[##                                                              ]  4.6615%
Alive[##                                                              ]  4.6745%
Alive[##                                                              ]  4.6875%
Alive[###                                                             ]  4.7005%
Alive[###                                                             ]  4.7135%
Alive[###                                                             ]  4.7266%
Alive[###                                                             ]  4.7396%
Alive[###                                                             ]  4.7526%
Alive[###                                                             ]  4.7656%
Alive[###                                                             ]  4.7786%
Alive[###                                                             ]  4.7917%
Alive[###                                                             ]  4.8047%
Alive[###                                                             ]  4.8177%
Alive[###                                                             ]  4.8307%
Alive[###                                                             ]  4.8437%
Alive[###                                                             ]  4.8568%
Alive[###                                                             ]  4.8698%
Alive[###                                                             ]  4.8828%
Alive[###                                                             ]  4.8958%
Alive[###                                                             ]  4.9089%
Alive[###                                                             ]  4.9219%
Alive[###                                                             ]  4.9349%
Alive[###                                                             ]  4.9479%
Alive[###                                                             ]  4.9609%
Alive[###                                                             ]  4.9740%
Alive[###                                                             ]  4.9870%
Alive[###                                                             ]  5.0000%
Alive[###                                                             ]  5.0130%
Alive[###                                                             ]  5.0260%
Alive[###                                                             ]  5.0391%
Alive[###                                                             ]  5.0521%
Alive[###                                                             ]  5.0651%
Alive[###                                                             ]  5.0781%
Alive[###                                                             ]  5.0911%
Alive[###                                                             ]  5.1042%
Alive[###                                                             ]  5.1172%
Alive[###                                                             ]  5.1302%
Alive[###                                                             ]  5.1432%
Alive[###                                                             ]  5.1562%
Alive[###                                                             ]  5.1693%
Alive[###                                                             ]  5.1823%
Alive[###                                                             ]  5.1953%
Alive[###                                                             ]  5.2083%
Alive[###                                                             ]  5.2214%
Alive[###                                                             ]  5.2344%
Alive[###                                                             ]  5.2474%
Alive[###                                                             ]  5.2604%
Alive[###                                                             ]  5.2734%
Alive[###                                                             ]  5.2865%
Alive[###                                                             ]  5.2995%
Alive[###                                                             ]  5.3125%
Alive[###                                                             ]  5.3255%
Alive[###                                                             ]  5.3385%
Alive[###                                                             ]  5.3516%
Alive[###                                                             ]  5.3646%
Alive[###                                                             ]  5.3776%
Alive[###                                                             ]  5.3906%
Alive[###                                                             ]  5.4036%
Alive[###                                                             ]  5.4167%
Alive[###                                                             ]  5.4297%
Alive[###                                                             ]  5.4427%
Alive[###                                                             ]  5.4557%
Alive[###                                                             ]  5.4687%
Alive[###                                                             ]  5.4818%
Alive[###                                                             ]  5.4948%
Alive[###                                                             ]  5.5078%
Alive[###                                                             ]  5.5208%
Alive[###                                                             ]  5.5339%
Alive[###                                                             ]  5.5469%
Alive[###                                                             ]  5.5599%
Alive[###                                                             ]  5.5729%
Alive[###                                                             ]  5.5859%
Alive[###                                                             ]  5.5990%
Alive[###                                                             ]  5.6120%
Alive[###                                                             ]  5.6250%
Alive[###                                                             ]  5.6380%
Alive[###                                                             ]  5.6510%
Alive[###                                                             ]  5.6641%
Alive[###                                                             ]  5.6771%
Alive[###                                                             ]  5.6901%
Alive[###                                                             ]  5.7031%
Alive[###                                                             ]  5.7161%
Alive[###                                                             ]  5.7292%
Alive[###                                                             ]  5.7422%
Alive[###                                                             ]  5.7552%
Alive[###                                                             ]  5.7682%
Alive[###                                                             ]  5.7812%
Alive[###                                                             ]  5.7943%
Alive[###                                                             ]  5.8073%
Alive[###                                                             ]  5.8203%
Alive[###                                                             ]  5.8333%
Alive[###                                                             ]  5.8464%
Alive[###                                                             ]  5.8594%
Alive[###                                                             ]  5.8724%
Alive[###                                                             ]  5.8854%
Alive[###                                                             ]  5.8984%
Alive[###                                                             ]  5.9115%
Alive[###                                                             ]  5.9245%
Alive[###                                                             ]  5.9375%
Alive[###                                                             ]  5.9505%
Alive[###                                                             ]  5.9635%
Alive[###                                                             ]  5.9766%
Alive[###                                                             ]  5.9896%
Alive[###                                                             ]  6.0026%
Alive[###                                                             ]  6.0156%
Alive[###                                                             ]  6.0286%
Alive[###                                                             ]  6.0417%
Alive[###                                                             ]  6.0547%
Alive[###                                                             ]  6.0677%
Alive[###                                                             ]  6.0807%
Alive[###                                                             ]  6.0937%
Alive[###                                                             ]  6.1068%
Alive[###                                                             ]  6.1198%
Alive[###                                                             ]  6.1328%
Alive[###                                                             ]  6.1458%
Alive[###                                                             ]  6.1589%
Alive[###                                                             ]  6.1719%
Alive[###                                                             ]  6.1849%
Alive[###                                                             ]  6.1979%
Alive[###                                                             ]  6.2109%
Alive[###                                                             ]  6.2240%
Alive[###                                                             ]  6.2370%
Alive[###                                                             ]  6.2500%
Alive[####                                                            ]  6.2630%
Alive[####                                                            ]  6.2760%
Alive[####                                                            ]  6.2891%
Alive[####                                                            ]  6.3021%
Alive[####                                                            ]  6.3151%
Alive[####                                                            ]  6.3281%
Alive[####                                                            ]  6.3411%
Alive[####                                                            ]  6.3542%
Alive[####                                                            ]  6.3672%
Alive[####                                                            ]  6.3802%
Alive[####                                                            ]  6.3932%
Alive[####                                                            ]  6.4062%
Alive[####                                                            ]  6.4193%
Alive[####                                                            ]  6.4323%
Alive[####                                                            ]  6.4453%
Alive[####                                                            ]  6.4583%
Alive[####                                                            ]  6.4714%
Alive[####                                                            ]  6.4844%
Alive[####                                                            ]  6.4974%
Alive[####                                                            ]  6.5104%
Alive[####                                                            ]  6.5234%
Alive[####                                                            ]  6.5365%
Alive[####                                                            ]  6.5495%
Alive[####                                                            ]  6.5625%
Alive[####                                                            ]  6.5755%
Alive[####                                                            ]  6.5885%
Alive[####                                                            ]  6.6016%
Alive[####                                                            ]  6.6146%
Alive[####                                                            ]  6.6276%
Alive[####                                                            ]  6.6406%
Alive[####                                                            ]  6.6536%
Alive[####                                                            ]  6.6667%
Alive[####                                                            ]  6.6797%
Alive[####                                                            ]  6.6927%
Alive[####                                                            ]  6.7057%
Alive[####                                                            ]  6.7187%
Alive[####                                                            ]  6.7318%
Alive[####                                                            ]  6.7448%
Alive[####                                                            ]  6.7578%
Alive[####                                                            ]  6.7708%
Alive[####                                                            ]  6.7839%
Alive[####                                                            ]  6.7969%
Alive[####                                                            ]  6.8099%
Alive[####                                                            ]  6.8229%
Alive[####                                                            ]  6.8359%
Alive[####                                                            ]  6.8490%
Alive[####                                                            ]  6.8620%
Alive[####                                                            ]  6.8750%
Alive[####                                                            ]  6.8880%
Alive[####                                                            ]  6.9010%
Alive[####                                                            ]  6.9141%
Alive[####                                                            ]  6.9271%
Alive[####                                                            ]  6.9401%
Alive[####                                                            ]  6.9531%
Alive[####                                                            ]  6.9661%
Alive[####                                                            ]  6.9792%
Alive[####                                                            ]  6.9922%
Alive[####                                                            ]  7.0052%
Alive[####                                                            ]  7.0182%
Alive[####                                                            ]  7.0312%
Alive[####                                                            ]  7.0443%
Alive[####                                                            ]  7.0573%
Alive[####                                                            ]  7.0703%
Alive[####                                                            ]  7.0833%
Alive[####                                                            ]  7.0964%
Alive[####                                                            ]  7.1094%
Alive[####                                                            ]  7.1224%
Alive[####                                                            ]  7.1354%
Alive[####                                                            ]  7.1484%
Alive[####                                                            ]  7.1615%
Alive[####                                                            ]  7.1745%
Alive[####                                                            ]  7.1875%
Alive[####                                                            ]  7.2005%
Alive[####                                                            ]  7.2135%
Alive[####                                                            ]  7.2266%
Alive[####                                                            ]  7.2396%
Alive[####                                                            ]  7.2526%
Alive[####                                                            ]  7.2656%
Alive[####                                                            ]  7.2786%
Alive[####                                                            ]  7.2917%
Alive[####                                                            ]  7.3047%
Alive[####                                                            ]  7.3177%
Alive[####                                                            ]  7.3307%
Alive[####                                                            ]  7.3437%
Alive[####                                                            ]  7.3568%
Alive[####                                                            ]  7.3698%
Alive[####                                                            ]  7.3828%
Alive[####                                                            ]  7.3958%
Alive[####                                                            ]  7.4089%
Alive[####                                                            ]  7.4219%
Alive[####                                                            ]  7.4349%
Alive[####                                                            ]  7.4479%
Alive[####                                                            ]  7.4609%
Alive[####                                                            ]  7.4740%
Alive[####                                                            ]  7.4870%
Alive[####                                                            ]  7.5000%
Alive[####                                                            ]  7.5130%
Alive[####                                                            ]  7.5260%
Alive[####                                                            ]  7.5391%
Alive[####                                                            ]  7.5521%
Alive[####                                                            ]  7.5651%
Alive[####                                                            ]  7.5781%
Alive[####                                                            ]  7.5911%
Alive[####                                                            ]  7.6042%
Alive[####                                                            ]  7.6172%
Alive[####                                                            ]  7.6302%
Alive[####                                                            ]  7.6432%
Alive[####                                                            ]  7.6562%
Alive[####                                                            ]  7.6693%
Alive[####                                                            ]  7.6823%
Alive[####                                                            ]  7.6953%
Alive[####                                                            ]  7.7083%
Alive[####                                                            ]  7.7214%
Alive[####                                                            ]  7.7344%
Alive[####                                                            ]  7.7474%
Alive[####                                                            ]  7.7604%
Alive[####                                                            ]  7.7734%
Alive[####                                                            ]  7.7865%
Alive[####                                                            ]  7.7995%
Alive[####                                                            ]  7.8125%
Alive[#####                                                           ]  7.8255%
Alive[#####                                                           ]  7.8385%
Alive[#####                                                           ]  7.8516%
Alive[#####                                                           ]  7.8646%
Alive[#####                                                           ]  7.8776%
Alive[#####                                                           ]  7.8906%
Alive[#####                                                           ]  7.9036%
Alive[#####                                                           ]  7.9167%
Alive[#####                                                           ]  7.9297%
Alive[#####                                                           ]  7.9427%
Alive[#####                                                           ]  7.9557%
Alive[#####                                                           ]  7.9687%
Alive[#####                                                           ]  7.9818%
Alive[#####                                                           ]  7.9948%
Alive[#####                                                           ]  8.0078%
Alive[#####                                                           ]  8.0208%
Alive[#####                                                           ]  8.0339%
Alive[#####                                                           ]  8.0469%
Alive[#####                                                           ]  8.0599%
Alive[#####                                                           ]  8.0729%
Alive[#####                                                           ]  8.0859%
Alive[#####                                                           ]  8.0990%
Alive[#####                                                           ]  8.1120%
Alive[#####                                                           ]  8.1250%
Alive[#####                                                           ]  8.1380%
Alive[#####                                                           ]  8.1510%
Alive[#####                                                           ]  8.1641%
Alive[#####                                                           ]  8.1771%
Alive[#####                                                           ]  8.1901%
Alive[#####                                                           ]  8.2031%
Alive[#####                                                           ]  8.2161%
Alive[#####                                                           ]  8.2292%
Alive[#####                                                           ]  8.2422%
Alive[#####                                                           ]  8.2552%
Alive[#####                                                           ]  8.2682%
Alive[#####                                                           ]  8.2812%
Alive[#####                                                           ]  8.2943%
Alive[#####                                                           ]  8.3073%
Alive[#####                                                           ]  8.3203%
Alive[#####                                                           ]  8.3333%
Alive[#####                                                           ]  8.3464%
Alive[#####                                                           ]  8.3594%
Alive[#####                                                           ]  8.3724%
Alive[#####                                                           ]  8.3854%
Alive[#####                                                           ]  8.3984%
Alive[#####                                                           ]  8.4115%
Alive[#####                                                           ]  8.4245%
Alive[#####                                                           ]  8.4375%
Alive[#####                                                           ]  8.4505%
Alive[#####                                                           ]  8.4635%
Alive[#####                                                           ]  8.4766%
Alive[#####                                                           ]  8.4896%
Alive[#####                                                           ]  8.5026%
Alive[#####                                                           ]  8.5156%
Alive[#####                                                           ]  8.5286%
Alive[#####                                                           ]  8.5417%
Alive[#####                                                           ]  8.5547%
Alive[#####                                                           ]  8.5677%
Alive[#####                                                           ]  8.5807%
Alive[#####                                                           ]  8.5937%
Alive[#####                                                           ]  8.6068%
Alive[#####                                                           ]  8.6198%
Alive[#####                                                           ]  8.6328%
Alive[#####                                                           ]  8.6458%
Alive[#####                                                           ]  8.6589%
Alive[#####                                                           ]  8.6719%
Alive[#####                                                           ]  8.6849%
Alive[#####                                                           ]  8.6979%
Alive[#####                                                           ]  8.7109%
Alive[#####                                                           ]  8.7240%
Alive[#####                                                           ]  8.7370%
Alive[#####                                                           ]  8.7500%
Alive[#####                                                           ]  8.7630%
Alive[#####                                                           ]  8.7760%
Alive[#####                                                           ]  8.7891%
Alive[#####                                                           ]  8.8021%
Alive[#####                                                           ]  8.8151%
Alive[#####                                                           ]  8.8281%
Alive[#####                                                           ]  8.8411%
Alive[#####                                                           ]  8.8542%
Alive[#####                                                           ]  8.8672%
Alive[#####                                                           ]  8.8802%
Alive[#####                                                           ]  8.8932%
Alive[#####                                                           ]  8.9062%
Alive[#####                                                           ]  8.9193%
Alive[#####                                                           ]  8.9323%
Alive[#####                                                           ]  8.9453%
Alive[#####                                                           ]  8.9583%
Alive[#####                                                           ]  8.9714%
Alive[#####                                                           ]  8.9844%
Alive[#####                                                           ]  8.9974%
Alive[#####                                                           ]  9.0104%
Alive[#####                                                           ]  9.0234%
Alive[#####                                                           ]  9.0365%
Alive[#####                                                           ]  9.0495%
Alive[#####                                                           ]  9.0625%
Alive[#####                                                           ]  9.0755%
Alive[#####                                                           ]  9.0885%
Alive[#####                                                           ]  9.1016%
Alive[#####                                                           ]  9.1146%
Alive[#####                                                           ]  9.1276%
Alive[#####                                                           ]  9.1406%
Alive[#####                                                           ]  9.1536%
Alive[#####                                                           ]  9.1667%
Alive[#####                                                           ]  9.1797%
Alive[#####                                                           ]  9.1927%
Alive[#####                                                           ]  9.2057%
Alive[#####                                                           ]  9.2187%
Alive[#####                                                           ]  9.2318%
Alive[#####                                                           ]  9.2448%
Alive[#####                                                           ]  9.2578%
Alive[#####                                                           ]  9.2708%
Alive[#####                                                           ]  9.2839%
Alive[#####                                                           ]  9.2969%
Alive[#####                                                           ]  9.3099%
Alive[#####                                                           ]  9.3229%
Alive[#####                                                           ]  9.3359%
Alive[#####                                                           ]  9.3490%
Alive[#####                                                           ]  9.3620%
Alive[#####                                                           ]  9.3750%
Alive[######                                                          ]  9.3880%
Alive[######                                                          ]  9.4010%
Alive[######                                                          ]  9.4141%
Alive[######                                                          ]  9.4271%
Alive[######                                                          ]  9.4401%
Alive[######                                                          ]  9.4531%
Alive[######                                                          ]  9.4661%
Alive[######                                                          ]  9.4792%
Alive[######                                                          ]  9.4922%
Alive[######                                                          ]  9.5052%
Alive[######                                                          ]  9.5182%
Alive[######                                                          ]  9.5312%
Alive[######                                                          ]  9.5443%
Alive[######                                                          ]  9.5573%
Alive[######                                                          ]  9.5703%
Alive[######                                                          ]  9.5833%
Alive[######                                                          ]  9.5964%
Alive[######                                                          ]  9.6094%
Alive[######                                                          ]  9.6224%
Alive[######                                                          ]  9.6354%
Alive[######                                                          ]  9.6484%
Alive[######                                                          ]  9.6615%
Alive[######                                                          ]  9.6745%
Alive[######                                                          ]  9.6875%
Alive[######                                                          ]  9.7005%
Alive[######                                                          ]  9.7135%
Alive[######                                                          ]  9.7266%
Alive[######                                                          ]  9.7396%
Alive[######                                                          ]  9.7526%
Alive[######                                                          ]  9.7656%
Alive[######                                                          ]  9.7786%
Alive[######                                                          ]  9.7917%
Alive[######                                                          ]  9.8047%
Alive[######                                                          ]  9.8177%
Alive[######                                                          ]  9.8307%
Alive[######                                                          ]  9.8437%
Alive[######                                                          ]  9.8568%
Alive[######                                                          ]  9.8698%
Alive[######                                                          ]  9.8828%
Alive[######                                                          ]  9.8958%
Alive[######                                                          ]  9.9089%
Alive[######                                                          ]  9.9219%
Alive[######                                                          ]  9.9349%
Alive[######                                                          ]  9.9479%
Alive[######                                                          ]  9.9609%
Alive[######                                                          ]  9.9740%
Alive[######                                                          ]  9.9870%
Alive[######                                                          ]  10.0000%
Alive[######                                                          ]  10.0130%
Alive[######                                                          ]  10.0260%
Alive[######                                                          ]  10.0391%
Alive[######                                                          ]  10.0521%
Alive[######                                                          ]  10.0651%
Alive[######                                                          ]  10.0781%
Alive[######                                                          ]  10.0911%
Alive[######                                                          ]  10.1042%
Alive[######                                                          ]  10.1172%
Alive[######                                                          ]  10.1302%
Alive[######                                                          ]  10.1432%
Alive[######                                                          ]  10.1562%
Alive[######                                                          ]  10.1693%
Alive[######                                                          ]  10.1823%
Alive[######                                                          ]  10.1953%
Alive[######                                                          ]  10.2083%
Alive[######                                                          ]  10.2214%
Alive[######                                                          ]  10.2344%
Alive[######                                                          ]  10.2474%
Alive[######                                                          ]  10.2604%
Alive[######                                                          ]  10.2734%
Alive[######                                                          ]  10.2865%
Alive[######                                                          ]  10.2995%
Alive[######                                                          ]  10.3125%
Alive[######                                                          ]  10.3255%
Alive[######                                                          ]  10.3385%
Alive[######                                                          ]  10.3516%
Alive[######                                                          ]  10.3646%
Alive[######                                                          ]  10.3776%
Alive[######                                                          ]  10.3906%
Alive[######                                                          ]  10.4036%
Alive[######                                                          ]  10.4167%
Alive[######                                                          ]  10.4297%
Alive[######                                                          ]  10.4427%
Alive[######                                                          ]  10.4557%
Alive[######                                                          ]  10.4687%
Alive[######                                                          ]  10.4818%
Alive[######                                                          ]  10.4948%
Alive[######                                                          ]  10.5078%
Alive[######                                                          ]  10.5208%
Alive[######                                                          ]  10.5339%
Alive[######                                                          ]  10.5469%
Alive[######                                                          ]  10.5599%
Alive[######                                                          ]  10.5729%
Alive[######                                                          ]  10.5859%
Alive[######                                                          ]  10.5990%
Alive[######                                                          ]  10.6120%
Alive[######                                                          ]  10.6250%
Alive[######                                                          ]  10.6380%
Alive[######                                                          ]  10.6510%
Alive[######                                                          ]  10.6641%
Alive[######                                                          ]  10.6771%
Alive[######                                                          ]  10.6901%
Alive[######                                                          ]  10.7031%
Alive[######                                                          ]  10.7161%
Alive[######                                                          ]  10.7292%
Alive[######                                                          ]  10.7422%
Alive[######                                                          ]  10.7552%
Alive[######                                                          ]  10.7682%
Alive[######                                                          ]  10.7812%
Alive[######                                                          ]  10.7943%
Alive[######                                                          ]  10.8073%
Alive[######                                                          ]  10.8203%
Alive[######                                                          ]  10.8333%
Alive[######                                                          ]  10.8464%
Alive[######                                                          ]  10.8594%
Alive[######                                                          ]  10.8724%
Alive[######                                                          ]  10.8854%
Alive[######                                                          ]  10.8984%
Alive[######                                                          ]  10.9115%
Alive[######                                                          ]  10.9245%
Alive[######                                                          ]  10.9375%
Alive[#######                                                         ]  10.9505%
Alive[#######                                                         ]  10.9635%
Alive[#######                                                         ]  10.9766%
Alive[#######                                                         ]  10.9896%
Alive[#######                                                         ]  11.0026%
Alive[#######                                                         ]  11.0156%
Alive[#######                                                         ]  11.0286%
Alive[#######                                                         ]  11.0417%
Alive[#######                                                         ]  11.0547%
Alive[#######                                                         ]  11.0677%
Alive[#######                                                         ]  11.0807%
Alive[#######                                                         ]  11.0937%
Alive[#######                                                         ]  11.1068%
Alive[#######                                                         ]  11.1198%
Alive[#######                                                         ]  11.1328%
Alive[#######                                                         ]  11.1458%
Alive[#######                                                         ]  11.1589%
Alive[#######                                                         ]  11.1719%
Alive[#######                                                         ]  11.1849%
Alive[#######                                                         ]  11.1979%
Alive[#######                                                         ]  11.2109%
Alive[#######                                                         ]  11.2240%
Alive[#######                                                         ]  11.2370%
Alive[#######                                                         ]  11.2500%
Alive[#######                                                         ]  11.2630%
Alive[#######                                                         ]  11.2760%
Alive[#######                                                         ]  11.2891%
Alive[#######                                                         ]  11.3021%
Alive[#######                                                         ]  11.3151%
Alive[#######                                                         ]  11.3281%
Alive[#######                                                         ]  11.3411%
Alive[#######                                                         ]  11.3542%
Alive[#######                                                         ]  11.3672%
Alive[#######                                                         ]  11.3802%
```

and I don't find that they bring any value 

```suggestion
    script_retry("curl -Sf https://build.suse.de/attribs/SUSE:Maintenance:$incident -o /tmp/$incident.txt");
```

Reply:
`-sSf`?